src/requests/matches.js

```diff
@@ -26,24 +26,30 @@ function parseMatch(xml) {
 	const id = xml.querySelector('MatchUniqueId')
 	const date = xml.querySelector('Date')
 	const time = xml.querySelector('Time')
-	const venue = xml.querySelector('Venue')
+	const venueEntry = xml.querySelector('VenueEntry')
 	const score = xml.querySelector('Score')
 	const system = xml.querySelector('MatchDetails MatchSystem')
 	const referee = xml.querySelector('MatchDetails Referee')
 	const division = xml.querySelector('DivisionName')
+	const divisionCategory = xml.querySelector('DivisionCategory')
```

Comment:
Division should be an object now as there are 3 properties: DivisionId, DivisionCategory, and DivisionName